test/internet/test-dns-ipv6.js

```diff
@@ -15,6 +15,17 @@ if (!common.hasIPv6) {
   return;
 }
 
+dns.lookup('localhost', {family: 6, all: true}, (err, addresses) => {
+  if (err) {
+    if (err.code === 'ENOTFOUND') {
+      common.skip('localhost does not resolve to ::1');
+      return;
+    }
+    throw err;
```

Comment:
Why is that? (Not trolling; honest question. If there's an `err` object, then that's basically an exception, not a test failure _per se_. Might be system DNS misconfiguration, DNS server unavailability, network problems...)